### PR TITLE
Add final report

### PR DESCRIPTION
**Pull request by jorgepiloto**
Adds the final report. Opening as draft for the moment 🚀

### CODE REVIEW

**Comment by jorgepiloto** [comment]
This is ready to review, let me now what you think @astrojuanlu and if some section needs to be developed a little bit more 👍🏽

File: report.md [hunk 1]
```diff
@@ -0,0 +1,149 @@
+---
+geometry: margin=2.5cm
+output: pdf_document
+colorlinks: true
+---
+
+<!-- Build PDF using pandoc: $ pandoc report.md -o report.pdf -->
+
+# Report for the NumFOCUS small development grants program
```
Comment (astrojuanlu) [inline report.md:9]: I would make a reference (in this title or in the first paragraph) to poliastro and the year of the funding (2020 3rd round)

File: report.md [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+## Obtained results and benefits
+
+Once all required poliastro's features were validated, the following results
+arose:
+
+* An error was detected in the Moon's rotational elements, which
+  which introduced numerical differences of the order of 1E-5. These proof the
+  frame validation logic to be successful.
+
+* The frame logic inside poliastro's API was proof to be right. This logic could
```
Comment (astrojuanlu) [inline report.md:94]: "proved" instead of proof?

File: report.md [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+* Auxiliary frames, such us planetary-fixed non-parallel to ICRF, can be
+  implemented.
+
+* Groundtrack plots can be now extended to the rest of planets of the Solar
+  System.
+
+
+## Associated pull-requests 
+
+All the pull-requests generated as a result of this work are listed down below,
```
Comment (astrojuanlu) [inline report.md:107]: No hyphen in pull-request, pull request is fine (here and elsewhere)

File: report.md [hunk 4]
```diff
@@ -0,0 +1,149 @@
...
+**poliastro repository**: https://github.com/poliastro/poliastro
+
+* [Fix MoonFrame W angle value:](https://github.com/poliastro/poliastro/pull/1193)
+  https://github.com/poliastro/poliastro/pull/1193
+
+* [Add GitHub actions for triggering validation repo:](https://github.com/poliastro/poliastro/pull/1051)
+  https://github.com/poliastro/poliastro/pull/1051
+
+* [Disable validation trigger in PR:](https://github.com/poliastro/poliastro/pull/1181)
+  https://github.com/poliastro/poliastro/pull/1181
```
Comment (astrojuanlu) [inline report.md:149]: Perhaps we can add a small paragraph with "future work"?

**Review by astrojuanlu — CHANGES_REQUESTED** [review]
This looks really great!! Added a couple of style comments, after that we can merge!

File: report.md (hunk 3 above)
Comment (jorgepiloto) [reply report.md:107]: Fixing this 🔧

File: report.md (hunk 2 above)
Comment (jorgepiloto) [reply report.md:94]: Ops 😓

File: report.md (hunk 1 above)
Comment (jorgepiloto) [reply report.md:9]: Good idea!

File: report.md (hunk 4 above)
Comment (jorgepiloto) [reply report.md:150]: Sure!

**Review by astrojuanlu — APPROVED** [review]
🌟 Awesome 🌟 Merging!

**Pull request merged**